chainer/training/extensions/plot_report.py

```diff
@@ -53,25 +54,27 @@ class PlotReport(extension.Extension):
         y_keys (iterable of strs): Keys of values regarded as y. If this is
             None, nothing is output to the graph.
         x_key (str): Keys of values regarded as x. The default value is
-            'iteration'
-        trigger: Trigger that decides when to aggregate the result and
-            output the values. This is distinct from the trigger of this
-            extension itself. If it is a tuple in the form
-            ``<int>, 'epoch'`` or ``<int>, 'iteration'``, it is passed to
-            :class:`IntervalTrigger`.
-        postprocess: Callback to postprocess the result dictionaries.
-            Figure object, Axes object, and all plot data is passed to this
-            callback on the output. This callback can modify the figure.
-        file_name (str): Name of the figure file under the output
-            directory. It can be a format string.
+            'iteration'.
+        trigger: Trigger that decides when to aggregate the result and output
+            the values. This is distinct from the trigger of this extension
+            itself. If it is a tuple in the form ``<int>, 'epoch'`` or ``<int>,
+            'iteration'``, it is passed to :class:`IntervalTrigger`.
+        postprocess: Callback to postprocess the result dictionaries. Figure
+            object, Axes object, and all plot data are passed to this callback
+            in this order. This callback can modify the figure.
+        file_name (str): Name of the figure file under the output directory.
+            It can be a format string.
 
     """
 
     def __init__(self, y_keys, x_key='iteration', trigger=(1, 'epoch'),
-                 postprocess=None, file_name='graph.png'):
+                 postprocess=None, file_name='plot.png'):
```

Comment:
It may not be a big deal, but I wonder whether we need to change the default file name here. Keeping the previous name to preserve backward compatibility may be better.